src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.spec.ts

```diff
@@ -96,6 +96,7 @@ describe('CoursePlayerComponent', () => {
     spyOn(courseConsumptionService, 'getCourseHierarchy').and.returnValue(of(CourseHierarchyGetMockResponse.result.content));
     component.ngOnInit();
     expect(component.courseHierarchy).toBeDefined();
+    expect(component.contributions).toBeDefined();
```

Comment:
test case to check where it is array or not like
`expect(_.isArray(component.contributions)).toBeTruthy();`